### ISSUE TITLE
Can't create account

### ISSUE THREAD

**Issue by hkk4685** [finndev/PokeBuddy#1086]
It says i need to create an account to log in,so i clicked create account,when i got in it, It says my browser is working,Cloudflare is working,but Host is in error,And this is what it says:
The initial connection between CloudFlare's network and the origin web server timed out. As a result, the web page can not be displayed.

i tried after a few minutes,but it still won't work,pls help

**Comment by yeapeh** [comment]
<img width="787" alt="error" src="https://cloud.githubusercontent.com/assets/20989887/17648088/22dcfac0-623c-11e6-822c-b420426e9c69.png">

same here

**Comment by GabrielAtanasov** [comment]
Yhea, site is currently unavailable

**Comment by hyryus** [comment]
same here

**Comment by GabrielAtanasov** [comment]
Can someone tries to DDOS site or developers have stopped for temporary maintenance

**Comment by ca2014mp2** [comment]
faaaaaaaaakkkkk

**Comment by Rayquazaaa** [comment]
We shouldn't even need a login for this bot.

**Comment by derherrmann** [comment]
I do not think it is a valid DDoS attack, I guess there are just too many requests by users who want to register and login, which also could lead to a denile of service.
